### PR TITLE
Added Eslint and Prettier configurations Fixes: #11

### PR DESCRIPTION
**Pull request by varunvme**
Basic configuration for Eslint and Perttier are implemented. **NOTE :** I've moved `App.js` to `/src` directory to keep all the `.js` rendering or providing component in a particular dir rather than root dir. And minor modifications of `' ' -> " "`(single/double quotes) or for `,`(comma) are due to prettier config which can be modified according to decisions made by maintainers.

### CODE REVIEW

**Review by sarthology — CHANGES_REQUESTED** [review]
Please delete this file, We are using yarn.

package-lock

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (sarthology) [inline .prettierrc]: Please add setting here

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import "react-native-gesture-handler";
```
Comment (sarthology) [inline src/App.js]: I knew changing this file will create issues. I suggest moving is back. It's giving errors

Screenshot 2020-03-01 at 4 08 33 AM

**Review by sarthology — CHANGES_REQUESTED** [review]
PLEASE, check my comments.

**Comment by varunvme** [comment]
As you directed me to move `App.js` back to root directory it's done and i kept `.prettierrc` blank inorder to not make the configurations opinionated to me, though i've pushed some generic and widely accepted configuration to it.

**Comment by sarthology** [comment]
It looks good know. Please resolve conflicts.

**Comment by sarthology** [comment]
Congratulations on your first successful PR. 🎉

**Comment by muskankhedia** [comment]
@sarthology Please do add the gssoc20 tag and easy, beginner, medium or hard tag before merging a PR as announced many times in the slack channel. Else many students may lose points because of this.

Please keep a note of this and also inform your mentors team for the smooth and successful conduct of the program.

**Pull request merged**